modules/default.nix

```diff
@@ -16,10 +16,12 @@ let
   rawModule = evalModules {
     modules = [
       {
+        _module.args.home-manager = home-manager;
         _module.args.pkgs = mkDefault pkgs;
+        _module.args.pkgsPath = pkgs.path;
```

Comment:
You don't need this, just use pkgs.path whereever you need it or do I miss something?